packages/sidebar/src/Sidebar.tsx

```diff
@@ -168,51 +157,46 @@ export const PureSidebar = ({
   return (
     <ThemeProvider theme={ theme || themeContext || {} }>
       <Root offsetLeft={ offsetLeft } isRTL={ isRTL }>
-        <SidebarWrapper
-          ref={ sidebarRef }
-          isRTL={ isRTL }
-        >
+        <SidebarWrapper ref={ sidebarRef } isRTL={ isRTL }>
           { navComponent && (
             <NavComponentWrapper>
-              <Scrollbar
-                width={ separatorWidth }
+              <Scrollbars
+                style={ { width: navWidth } }
                 autoHide={ showScrollbar === 'auto' }
-                animate={ false }
               >
                 { navComponent }
-              </Scrollbar>
+              </Scrollbars>
             </NavComponentWrapper>
           ) }
           { children && (
             <ResponsiveWrapper
-              animate={ transformParams.animate }
-              style={ responsiveWrapperStyles }
+              animate={ animate }
+              style={ { width } }
               isRTL={ isRTL }
             >
-              <ChildWrapper
-                style={ { width: transformParams.width } }
-                animate={ transformParams.animate }
-              >
-                <Scrollbar
-                  animate={ transformParams.animate }
-                  width={ rightWidth }
-                  marginLeft="-10px"
+              <ChildWrapper animate={ animate }>
+                <Scrollbars
+                  style={ {
```

Comment:
There are some problems: 
1) Here Scrollbars will be taking a new object of style in every render cycle
2) The style attribute is not god when styled-components is used.  https://styled-components.com/docs/api#attrs

Reply:
If wrap Scrollbars inside styled we'll get console error about passing unrecognized props inside DOM element.
In current situation width always will change so there are no reason wrap it in useMemo.
And that is why current solution is OK.